### PR TITLE
MTA assessment application review

### PR DESCRIPTION
**Pull request by HagayVider1**
Replaced by https://github.com/windup/windup-documentation/pull/799

### CODE REVIEW

File: docs/topics/mta-assessment-app-review.adoc
Comment (HagayVider1) [inline docs/topics/mta-assessment-app-review.adoc:1]: File not relevant. Remove
All information in other file.

File: docs/topics/mta-assessment-app-review.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
++
+// Get updated image for MTA
+image::mta-assessment-questionnaire-review-01.png[Assessment questionnaire]
++
+
+. Click *Proposed action* and select the action.
+. Click *Effort estimate* and set the level of effort required to perform the assessment with the selected questionnaire.
+. In the *Business criticality* field enter how critical the application is to the business.
+. In the *Work priority* field enter the application's priority.
+. If you have any comments for the questionnaire, then enter them in the *Comments* field.
```
Comment (emarcusRH) [inline docs/topics/mta-assessment-app-review.adoc:31]:
```suggestion
. If you have any comments for the questionnaire, enter them in the *Comments* field.
```

File: docs/topics/mta-web-reviewing-apps.adoc [hunk 2]
```diff
@@ -11,16 +11,21 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the mi
...
+. Click image:mta-web-icon-actions.png[] (three dots) at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
+
+// Get updated image for MTA
+image::mta-web-assessment-review-01.png[Application review]
+
+. Click *Proposed action* and select the action.
+. Click *Effort estimate* and set the level of effort required to perform the assessment with the selected questionnaire.
+. In the *Business criticality* field enter how critical the application is to the business.
+. In the *Work priority* field enter the application's priority.
+. If you have any comments for the questionnaire, then enter them in the *Comments* field.
```
Comment (emarcusRH) [inline docs/topics/mta-web-reviewing-apps.adoc:28]:
```suggestion
. If you have any comments for the questionnaire, enter them in the *Comments* field.
```

**Comment by HagayVider1** [comment]
Not relevant
Replaced by https://github.com/windup/windup-documentation/pull/799

File: docs/topics/mta-web-reviewing-apps.adoc [hunk 3]
```diff
@@ -11,16 +11,21 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the mi
...
 . Select the application you want to review.
-+
 [NOTE]
 ====
 Only one application can be reviewed at a time.
 ====
-. Click  *Review*.
-. Select *Proposed action* and *Effort estimate* from the lists. Set *Business criticality* and *Work priority*.
-. Click *Submit review*.
+. Click image:kebab.png[] (three dots) at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
```
Comment (anarnold97) [inline docs/topics/mta-web-reviewing-apps.adoc:19]:
```suggestion
. Click the Options menu {kebab} at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
```

**Comment by jmle** [comment]
Replaced by https://github.com/windup/windup-documentation/pull/799

**Pull request closed without merging**